config/routes.rb

```diff
@@ -10,6 +10,10 @@
   resources :categories
   resources :images
 
```

Comment:
perhaps I could scope out more in the routes so that its impossible for other users to even access that.

Reply:
asked an upper mod and he said not to really do that just yet

Reply:
be specific in your routes.